airlock/forms.py

```diff
@@ -16,7 +16,7 @@ class AddFileForm(forms.Form):
         choices=[
             (i.name, i.name.title())
             for i in RequestFileType
-            if i != RequestFileType.WITHDRAWN
+            if i not in [RequestFileType.WITHDRAWN, RequestFileType.CODE]
```

Comment:
should we just allow-list?

```suggestion
            if i in [RequestFileType.OUTPUT, RequestFileType.SUPPORTING]
```

Reply:
Maybe? I thought about it, wasn't sure if there could ever be another type in future. There's a test now at least, if we did this again

Reply:
🤷 I wouldn't have expected CODE to be added tbh, but not really fussed.